example_test.go

```diff
@@ -252,3 +280,24 @@ func ExampleCopy_localToRemote() {
 	// Output:
 	// sha256:6f2590d54af17afaca41a6243e3c01b368117d24b32e7581a6dee1d856dd3c4b
 }
+
+func ExampleCopy_artifactManifestRemoteToLocal() {
+	src, err := remote.NewRepository(fmt.Sprintf("%s/%s", remoteHost, "source"))
+	if err != nil {
+		panic(err)
+	}
+	dst := memory.New()
+
+	ctx := context.Background()
+	err = oras.CopyGraph(ctx, src, dst, exampleSignatureManifestDescriptor, oras.DefaultCopyGraphOptions)
+	if err != nil {
+		panic(err)
+	}
+
+	// verify that the content of dst is described by exampleSignatureManifestDescriptor
+	contentExists, err := dst.Exists(ctx, exampleSignatureManifestDescriptor)
+	fmt.Println(contentExists)
+
```

Comment:
I think we need a better way to verify the copied content and its subject, but I don't know how.

Reply:
We can follow the same pattern as other examples.